cucumber-android/src/main/java/cucumber/runtime/android/CucumberJUnitRunnerBuilder.java

```diff
@@ -0,0 +1,15 @@
+package cucumber.runtime.android;
+
+import org.junit.runner.Runner;
+import org.junit.runners.model.RunnerBuilder;
+
+public class CucumberJUnitRunnerBuilder extends RunnerBuilder {
+    @Override
+    public Runner runnerForClass(final Class<?> testClass) {
+        if (testClass.equals(this.getClass())) {
```

Comment:
it is needed for this Builder to work only once - so we run it one Class from tests Classpath - for itself